stdlib/2and3/aifc.pyi

```diff
@@ -0,0 +1,81 @@
+
+from typing import Union, IO, Optional, Type, NamedTuple, List, Tuple, Any, Text, overload
+from typing_extensions import Literal
+from types import TracebackType
+import sys
+
+class Error(Exception): ...
+
+_aifc_params = NamedTuple('_aifc_params', [('nchannels', int), ('sampwidth', int), ('framerate', int),
+                                           ('nframes', int), ('comptype', bytes), ('compname', bytes)])
+
+_File = Union[Text, IO[bytes]]
+_Marker = Tuple[int, int, bytes]
+
+class Aifc_read:
+    def __init__(self, f: _File) -> None: ...
+    if sys.version_info >= (3, 4):
+        def __enter__(self) -> Aifc_read: ...
+        def __exit__(self, exc_type: Optional[Type[BaseException]], exc_val: Optional[BaseException],
+                     exc_tb: Optional[TracebackType]) -> None: ...
+    def initfp(self, file: IO[bytes]) -> None: ...
+    def getfp(self) -> IO[bytes]: ...
+    def rewind(self) -> None: ...
+    def close(self) -> None: ...
+    def tell(self) -> int: ...
+    def getnchannels(self) -> int: ...
+    def getnframes(self) -> int: ...
+    def getsampwidth(self) -> int: ...
+    def getframerate(self) -> int: ...
+    def getcomptype(self) -> bytes: ...
+    def getcompname(self) -> bytes: ...
+    def getparams(self) -> _aifc_params: ...
+    def getmarkers(self) -> Optional[List[_Marker]]: ...
+    def getmark(self, id: int) -> _Marker: ...
+    def setpos(self, pos: int) -> None: ...
+    def readframes(self, nframes: int) -> str: ...
```

Comment:
Shouldn't this return `bytes`? At least that's what return when `nframes == 0` and it would make more sense to me.

Reply:
`bytes` does seem to make more sense to me. It has been long enough that I'm not sure my logic in making it a `str`